.github/workflows/build.yml

```diff
@@ -148,6 +174,10 @@ jobs:
         with:
           name: lifecycle-linux-arm64-sha256
           path: out/lifecycle-v*+linux.arm64.tgz.sha256
+      - uses: actions/upload-artifact@v2
+        with:
+          name: lifecycle-freebsd-x86-64
+          path: out/lifecycle-v*+freebsd.x86-64.tgz
```

Comment:
Is the intent also to make the artifact available on releases? If so we'll also need to update draft-release.yml (for attaching the artifact to the release), post-release.yml (to re-tag images from Git commit to semver), and around line 236 in this file to publish the Git-tagged image.

Reply:
The last commits should do it 😃